<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TexasWidget</class>
 <widget class="QWidget" name="TexasWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TexasWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:gray;</string>
  </property>
  <widget class="QLabel" name="labelDealer">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resources/images/dealer.svg</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBet">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelPerflop1User1">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>150</y>
     <width>74</width>
     <height>105</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>74</width>
     <height>105</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>74</width>
     <height>105</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resources/images/back.svg</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>501</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelFlop1">
      <property name="minimumSize">
       <size>
        <width>74</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>74</width>
        <height>105</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/resources/images/back.svg</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="spacerhflop1">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="labelFlop2">
      <property name="minimumSize">
       <size>
        <width>74</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>74</width>
        <height>105</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/resources/images/back.svg</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="spacerhflop2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="labelFlop3">
      <property name="minimumSize">
       <size>
        <width>74</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>74</width>
        <height>105</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/resources/images/back.svg</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="spacerhflop3">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="labelTurn">
      <property name="minimumSize">
       <size>
        <width>74</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>74</width>
        <height>105</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/resources/images/back.svg</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="spacerhflop4">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="labelRiver">
      <property name="minimumSize">
       <size>
        <width>74</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>74</width>
        <height>105</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/resources/images/back.svg</pixmap>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelPerflop2User1">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>150</y>
     <width>74</width>
     <height>105</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>74</width>
     <height>105</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>74</width>
     <height>105</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resources/images/back.svg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelUser1">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>30</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resources/images/player_woman.svg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelPerflop2User2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>480</y>
     <width>74</width>
     <height>105</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>74</width>
     <height>105</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>74</width>
     <height>105</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resources/images/back.svg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelPerflop1User2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>480</y>
     <width>74</width>
     <height>105</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>74</width>
     <height>105</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>74</width>
     <height>105</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resources/images/back.svg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelUser2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>610</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resources/images/player_man.svg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelPerflop2User3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>74</width>
     <height>105</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>74</width>
     <height>105</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>74</width>
     <height>105</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resources/images/back.svg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelUser3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resources/images/player_woman.svg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelPerflop1User3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>74</width>
     <height>105</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>74</width>
     <height>105</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>74</width>
     <height>105</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resources/images/back.svg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelBet">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>当前筹码：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>680</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBet">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>610</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下注</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCheck">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>650</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>让牌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFold">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>690</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>弃牌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRaise">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>630</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>加注</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCall">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>670</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>跟注</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
